TS_src/client/src/App.tsx

```diff
@@ -6,7 +6,7 @@ import Login from './components/pages/Login';
 import Registration from './components/pages/Registration';
 import Dashboard from './components/pages/Dashboard';
 import ChatPage from './components/ChatPage';
-import { User } from './components/types';
+import { User, Trip } from './components/types';
 import socket from './main';
 import toast, { Toaster } from 'react-hot-toast';
```

Comment:
Is the import `toast` being used? I see that Toaster is for sure, but wasn't sure about toast. I noticed there were other unused imports as well such as `{ Component }` from `'react'` and `{ Link }` from `react-router-dom`. Are those safe to take out?

Reply:
Removed